channeldb/kvdb/backend.go

```diff
@@ -21,7 +21,9 @@ func fileExists(path string) bool {
 
 // GetBoltBackend opens (or creates if doesn't exits) a bbolt
 // backed database and returns a kvdb.Backend wrapping it.
-func GetBoltBackend(path, name string, noFreeListSync bool) (Backend, error) {
+func GetBoltBackend(path, name string, noFreeListSync,
```

Comment:
May be time to split this out into its own config as once we update to the latest version of `btcwallet`, there'll be a new option for opening the DB with a timeout.